src/main/java/Event.java

```diff
@@ -0,0 +1,19 @@
+public class Event extends Task {
```

Comment:
Excellent use of OOP here, and its great that the "type" variable was correctly used in the toString() method.